src/frontends/pytorch/src/op/aten_to.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "op_table.hpp"
+
+#include "pt_framework_node.hpp"
+
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+const std::map<int, element::Type> type_map{
+    {0, element::u8},
+    {1, element::i8},
+    {2, element::i16},
+    {3, element::i32},
+    {4, element::i64},
+    {5, element::f16},
+    {6, element::f32},
+    {7, element::f64},
+};
+
+
+OutputVector translate_aten_to(NodeContext& context) {
+    int dtype_idx;
+    int non_blocking_idx;
+    int copy_idx;
+    int memory_format_idx;
+    if (context.get_input_size() == 5) {
+        // aten::to.dtype(Tensor(a) self, int dtype, bool non_blocking=False, bool copy=False, int? memory_format=None)
+        // -> (Tensor(a))
+        dtype_idx = 1;
+        non_blocking_idx = 2;
+        copy_idx = 3;
+        memory_format_idx = 4;
+    } else if (context.get_input_size() == 6) {
+        // aten::to.device(Tensor(a) self, Device device, int dtype, bool non_blocking=False, bool copy=False, int?
+        // memory_format=None) -> (Tensor(a)).
+        // Skipping "device" input.
+        dtype_idx = 2;
+        non_blocking_idx = 3;
+        copy_idx = 4;
+        memory_format_idx = 5;
+    } else {
+        FRONT_END_OP_CONVERSION_CHECK(false, "Unknown aten::to format");
+    }
+    // TODO: do we need to check these inputs?
+    // OV_FRONTEND_REQUIRE(context.const_input<bool>(non_blocking_idx) == false);
+    // OV_FRONTEND_REQUIRE(context.const_input<bool>(copy_idx) == false);
+    // OV_FRONTEND_REQUIRE(context.input_is_none(memory_format_idx));
```

Comment:
Please remove all commented code. Please also take a look at what I did in my PR for `aten::to` translator (https://github.com/slyalin/openvino/pull/26/files#diff-a0034b0a7eb1e37f896730531219d6ceae6a2723700cd8e20dd5ee98937186f0). It would be nice to add comments for each of skipped inputs as to what they do and why do we think we can amend them.  Maybe also cover those inputs with layer tests.